docs/src/FlexBox.doc.js

```diff
@@ -0,0 +1,173 @@
+      {
+        name: 'flex',
+        type: '"grow" | "shrink" | "none"',
+        defaultValue: 'shrink',
+        description: `Defines how a flex item will be sized. "grow", equivalent to "flex: 1 1 auto", will size the FlexBox relative to its parent, growing and shrinking based on available space. "shrink", equivalent to "flex: 0 1 auto" (the browser default), allows the FlexBox to shrink if compressed but not grow if given extra space. Finally, "none", equivalent to "flex: 0 0 auto", preserves the FlexBox's size based on child content regardless of its container's size.`,
+      },
+      {
+        name: 'justifyContent',
+        type: `"start" | "end" | "center" | "between" | "around" | "evenly"`,
+        defaultValue: 'start',
+        description:
+          'Defines the alignment along the main axis. It helps distribute extra free space left over when either all the flex items on a line are inflexible, or are flexible but have reached their maximum size. It also exerts some control over the alignment of items when they overflow the line.',
+        href: 'layout',
+      },
+      {
+        name: 'maxHeight',
+        type: `number | string`,
+      },
+      {
+        name: 'maxWidth',
+        type: `number | string`,
+        description: `Use numbers for pixels: maxWidth={100} and strings for percentages: maxWidth="100%"`,
+      },
+      {
+        name: 'minHeight',
+        type: `number | string`,
+        description: `Use numbers for pixels: minHeight={100} and strings for percentages: minHeight="100%"`,
+      },
+      {
+        name: 'minWidth',
+        type: `number | string`,
+        description: `Use numbers for pixels: minWidth={100} and strings for percentages: minWidth="100%"`,
+      },
+      {
+        name: 'height',
+        type: `number | string`,
+        description: `Use numbers for pixels: height={100} and strings for percentages: height="100%"`,
+      },
+      {
+        name: 'width',
+        type: `number | string`,
+        description: `Use numbers for pixels: width={100} and strings for percentages: width="100%"`,
+      },
+      {
+        name: 'overflow',
+        type: `"visible" | "hidden" | "scroll" | "scrollX" | "scrollY" | "auto"`,
+        defaultValue: 'visible',
+      },
+      ...paddingProps,
+      {
+        name: 'wrap',
+        type: 'boolean',
+        defaultValue: false,
+        description: `By default, flex items will all try to fit onto one line. You can change that and allow the items to wrap onto multiple lines, from top to bottom.`,
+      },
+    ]}
+  />
+);
+
+card(
+  <Example
+    description={`
+    With a limited set of props that only relate to layout, FlexBox is useful for separating concerns to prevent overloaded Box usage.
+    Compare this example to the first example in the Box docs.
+  `}
+    name="Example: Media object"
+    defaultCode={`
+<FlexBox
+  alignItems="center"
+  marginStart={-1}
```

Comment:
@dangerismycat is this a prop supported by `FlexBox`? If so, could we add it to the docs.